source/docs/app/templates/nav.html

```diff
@@ -1,4 +1,4 @@
-<nav class="navbar navbar-default navbar-fixed-top" collapsing-navbar>
+<nav class="navbar navbar-default navbar-fixed-top" collapsing-navbar="true">
```

Comment:
#1stworldproblems: I'm not too enthusiastic about that boolean param here. How about we wrap this in an optional config object param, which extends/overrides a default directive config object?